internal/namespaces/autocomplete/autocomplete.go

```diff
@@ -83,7 +83,7 @@ var autocompleteScripts = map[string]autocompleteScript{
 		CompleteFunc: `
 			complete --erase --command scw;
 			complete --command scw --no-files;
-			complete --command scw --arguments '(scw autocomplete complete fish -- (commandline) (commandline --cursor) (commandline --current-token) (commandline --tokenize --cut-at-cursor))';
+			complete --command scw --arguments '(scw autocomplete complete fish -- (commandline) (commandline --cursor) (commandline --current-token) (commandline --current-process --tokenize --cut-at-cursor))';
```

Comment:
do we need `--current-process` flag  for other  `commandline` command ?

Reply:
`(commandline)` and `(commandline --cursor)` are not used yet. If we want to use them they must remain as is because the `cursor` points to the character index on the whole buffer ( returned by `commandline`)